subsys/CMakeLists.txt

```diff
@@ -10,3 +10,5 @@ add_subdirectory_ifdef(CONFIG_MCUBOOT_IMG_MANAGER  dfu)
 add_subdirectory_ifdef(CONFIG_NET_BUF              net)
 add_subdirectory_ifdef(CONFIG_USB                  usb)
 add_subdirectory(random)
+add_subdirectory(storage)
+add_subdirectory_ifdef(CONFIG_FCB  fs/fcb)
```

Comment:
Have you kept this out of the fs directory because FCB does not depend on CONFIG_FILE_SYSTEM?

We need to find some way of getting it into fs as users will expect this code to be in there. If FCB does not require CONFIG_FILE_SYSTEM you will need to refactor away the ifdef FILE_SYSTEM in this file. PM me if I am being unclear.

Reply:
>Have you kept this out of the fs directory because FCB does not depend on CONFIG_FILE_SYSTEM?

Yes, exactly.

Reply:
FCB code do not depend on CONFIG_FILE_SYSTEM already.